libs/api-spec/api-spec.yml

```diff
@@ -987,6 +1015,21 @@ components:
         service_done_at:
           type: string
   responses:
+    200_Dashboard:
+      description: Example response
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: '#/components/schemas/Meta_200OK'
+              - type: object
+                properties:
+                  data:
+                    type: object
+                    properties:
+                      total:
+                        type: string

```

Comment:
prefer to change it into `number` or `integer`

Reply:
ohhh, my bad

Reply:
done https://github.com/ALTERA-CAPSTON-41/backend/pull/45/commits/d25c353f51a81b4f48af42a4bd511325bf5ffe26